godot-core/src/obj/gd.rs

```diff
@@ -569,6 +571,11 @@ where
     }
 }
 
+/// # Safety
+/// `Gd<T: GodotClass>` will only contain types that inherit from `crate::engine::Object`.
+/// Godots `Object in turn is known to be nullable and always a pointer.
+unsafe impl<T: GodotClass> GodotNullablePtr for Gd<T> {}
```

Comment:
The convention for _usage_ of `unsafe` (as opposed to _declaration_) is a regular comment with `SAFETY:`, not a doc comment:
```
// SAFETY:
```
See other examples in the codebase.